src/shards/JLLEnvs.jl

```diff
@@ -3,18 +3,41 @@ module JLLEnvs
 using Pkg
 using Pkg.Artifacts
 using Downloads
+using UUIDs
 
 include("utils.jl")
 
 const JLL_ENV_SHARDS_URL = "https://raw.githubusercontent.com/JuliaPackaging/BinaryBuilderBase.jl/master/Artifacts.toml"
 const JLL_ENV_SHARDS = Dict{String,Any}()
 
+function _checked_import(pkgid)
+    Base.root_module_exists(pkgid) && return Base.root_module(pkgid)
+    Base.require(pkgid)
+end
+
 function __init__()
     if haskey(ENV, "JULIA_CLANG_SHARDS_URL") && !isempty(get(ENV, "JULIA_CLANG_SHARDS_URL", ""))
         merge!(JLL_ENV_SHARDS, Artifacts.load_artifacts_toml(ENV["JULIA_CLANG_SHARDS_URL"]))
-    else
-        merge!(JLL_ENV_SHARDS, Artifacts.load_artifacts_toml(Downloads.download(JLL_ENV_SHARDS_URL)))
+        return
     end
```

Comment:
Still keep the `JULIA_CLANG_SHARDS_URL` environment flag of the highest priority so that people can still debug with specific Artifacts.toml version.